eql/functions.py

```diff
@@ -322,17 +210,29 @@ def callback(source, *_):
     @classmethod
     def run(cls, ip_address, *cidr_matches):
         """Compare an IP address against a list of cidr blocks."""
-        if is_string(ip_address) and cls.ip_compiled.match(ip_address):
-            ip_integer, _ = cls.to_mask(ip_address + "/32")
+        if is_string(ip_address):
+            ip_address = get_ipaddress(ip_address)
 
             for cidr in cidr_matches:
-                if is_string(cidr) and cls.cidr_compiled.match(cidr):
-                    subnet, mask = cls.to_mask(cidr)
-                    if ip_integer & mask == subnet:
+                if is_string(cidr):
+                    subnet = get_subnet(cidr)
+
+                    if ip_address in subnet:
                         return True
 
         return False
 
+    @classmethod
+    def is_cidr(cls, cidr):
```

Comment:
```suggestion
    def is_cidr_pattern(cls, cidr):
```

or `is_cidr_notation`

we can also move this to utils as well I would think